docs/using/alerting.md

```diff
@@ -1,182 +1,339 @@
+              summary: MongoDB high memory usage ({{ $labels.service_name }})
+              description: |-
+                 {{ $value }}% of memory (more than [[ .threshold ]]%) is used
+                 by {{ $labels.service_name }} on {{ $labels.node_name }}.
+    ```
+## Test alert expressions
+If you want to create custom templates, you can test the MetricsQ expressions for your custom template in the **Explore** section of PMM. Here you can also query any PMM internal database.
+ 
+To test expressions for custom templates:
+1. On the left menu of PMM, choose **Explore > Metrics**.
+2. Enter your expression in the **Metrics** field and click **Run query**.
+ 
+For example, to validate that a MongoDB instance is down, shut down a member of a three-node replica set, then check that the expression **{service_type="mongodb"}**  returns **0** in **Explore > Metrics**.
+ 
+## Template compatibility
+### With previous PMM versions
+PMM 2.26 introduced major changes to the core structure of rule templates. As a result, alert rules and templates created in previous PMM versions are not compatible with PMM 2.26.0 and cannot be migrated to this new version.
+ 
+If you’re migrating from PMM 2.25 and earlier, make sure to manually recreate any custom alert rules and rule templates that you want to transfer to PMM 2.26.0 or later.
+ 
+### With other alerting tools
+ 
+If you have existing YAML alert templates that you want to leverage in PMM Integrated Alerting:
+1. Go to **Alerting > Integrated Alerting > Alert Rule Templates** tab and click **Add** at the top right-hand side of the table.
+2. Click **Add** and upload a local .yaml file from your computer.
+ 
 ## Add an Alert Rule
-
-1. Select the *Alert Rules* tab.
-
-    ![!](../_images/PMM_Integrated_Alerting_Alert_Rules.jpg)
-
-1. Click <i class="uil uil-plus-square"></i> *Add*.
-
-1. Fill in the details
-
-    ![!](../_images/PMM_Integrated_Alerting_Alert_Rules_Add_Form.jpg)
-
-    - Template:
-    - Name:
-    - Threshold:
-    - Duration(s):
-    - Severity:
-    - Filters:
-    - Channels:
-    - Activate:
-
-1. Click *Add* to add the alert rule, or *Cancel* to abort the operation.
-
-## Add an Alert Rule Template
-
-If the provided alert rule templates don't do what you want, you can create your own.
-
-1. Select the *Alert Rule Templates* tab.
-
-    ![!](../_images/PMM_Integrated_Alerting_Alert_Rule_Templates.jpg)
-
-1. Click <i class="uil uil-plus-square"></i> *Add*.
-
-1. Enter a template in the *Alert Rule Template* text box.
-
+1. Go to **Alerting > Integrated Alerting** and select the **Alert Rules** tab.
+2. Click **Add** on the right-hand side of the table.
+3. In the **Add Alert** rule dialog, choose the template on which you want to base the new alert rule. This automatically populates the **Threshold**, **Duration**, **Severity** fields with information from the template. You can change these values if you want to override the default specifications in the template.
```

Comment:
`Threshold` and/or others is a parameter coming from the template defined by the user, so we shouldn't talk about it here. The fields that show up are the params defined beforehand (but Duration and Severity are indeed always there).